### PR TITLE
update app with 4. get article by article_id

### PR DESCRIPTION
**Pull request by benboosh01**
Not sure what has caused the conflicts so any advice welcome! :)

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+const request = require('supertest');
+const app = require('../app');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+testData = require('../db/data/test-data/index');
```
Comment (popatre) [inline __tests__/app.test.js:5]: `const`

File: __tests__/app.test.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+describe('GET /api/articles/:article_id', () => {
+  test('status 200: responds with object with correct keys for specific article id', () => {
+    return request(app)
+      .get('/api/articles/1')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toHaveProperty('author');
+        expect(body.article).toHaveProperty('title');
+        expect(body.article).toHaveProperty('article_id');
```
Comment (popatre) [inline __tests__/app.test.js:38]: need to check you're getting the id you requested - this will pass even if you get the wrong one

**Review by popatre — APPROVED** [review]
Just a small change to that test please, then free to merge

File: __tests__/app.test.js (hunk 2 above)
Comment (benboosh01) [reply __tests__/app.test.js]: fixed

**Pull request merged**